tests/integration/test_slicing.py

```diff
@@ -60,27 +56,44 @@ def test():
     print(dnp)
     print(d)
 
-    natest = np.random.random((2, 3, 4))
-    natestg = num.array(natest)
+
+NATEST = np.random.random((2, 3, 4))
+
+
+def test_3d():
+    natestg = num.array(NATEST)
 
     firstslice = natestg[0]
-    firstslicegold = natest[0]
+    firstslicegold = NATEST[0]
     assert np.array_equal(firstslice, firstslicegold)
 
-    # TODO: Legate needs 4-D arrays for this to work correctly
-    # secondslice = natestg[:,np.newaxis,:,:]
-    # secondslicegold = natest[:,np.newaxis,:,:]
-    # assert(num.array_equal(secondslice, secondslicegold))
 
-    # TODO: Legate needs 4-D arrays for this to work correctly
-    # thirdslice = natestg[np.newaxis]
-    # thirdslicegold = natest[np.newaxis]
-    # print(thirdslice)
-    # print(thirdslicegold)
-    # assert(num.array_equal(thirdslice, thirdslicegold))
+# TODO: Legate needs 4-D arrays for this to work correctly
+@pytest.mark.skip
+def test_4d():
+    natestg = num.array(NATEST)
+
+    secondslice = natestg[:, np.newaxis, :, :]
+    secondslicegold = NATEST[:, np.newaxis, :, :]
+    assert num.array_equal(secondslice, secondslicegold)
 
-    return
+    thirdslice = natestg[np.newaxis]
+    thirdslicegold = NATEST[np.newaxis]
+    assert num.array_equal(thirdslice, thirdslicegold)
+
+
+# TODO: Legate needs partition by phase for this to work efficiently
+@pytest.mark.skip
+def test_slice_step():
+    x = num.array(
+        [[1, 2, 3, 4], [5, 6, 7, 8], [9, 10, 11, 12], [13, 14, 15, 16]]
+    )
+    z = x[1:4:2, 1:4:2]
+    assert num.array_equal(z[0, :], [6, 8])
+    assert num.array_equal(z[1, :], [14, 16])
```

Comment:
Example of commented code moved under a `pytest.mark.skip` Pytest also supports conditional skipping, and also marking known/expected failures.